README_ja-JP.md

```
@@ -22,9 +30,35 @@
+_Legion Series Discord に参加: https://discord.com/invite/legionseries_<br>
+_LOQ Series Discord に参加: https://discord.gg/3GKzQtwdNf_
+
+# ローカライズ
+このフォークは Crowdin の [LenovoLegionToolkit](https://crowdin.com/project/lenovolegiontoolkit-unofficial) に連携しています。
+
+ローカライズへの貢献を歓迎し、深く感謝します！
+
+# スクリーンショット
+
+### メインウィンドウ
+<img src="assets/Main_DarkMode.png" width="49%" /> <img src="assets/Main_LightMode.png" width="49%" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)